docs/it/metadata_oidc_rp.rst

```diff
@@ -96,6 +96,9 @@ Il Metadata di tipo **"openid_relying_party"** DEVE contenere almeno i seguenti
   * - **client_registration_types**
     - Vedi `OIDC-FED`_ Section 4.1. Il valore richiesto è **automatic**. 
     - |spid-icon| |cieid-icon|
+  * - **response_types**
+    - Array dei valori di response_type previsti da OAuth 2.0 che il RP userà nelle richieste di autenticazione. Deve contenere il solo valore **code**. 
+    - |spid-icon| |cieid-icon|
 
 .. note:: 
   Gli URI presenti nel parametro **redirect_uris** POSSONO anche seguire eventuali schemi custom (ad es. myapp://) al fine di supportare applicazioni mobili.
```

Comment:
seguire -> usare/adottare

Reply:
Si vuole evitare che il "response_types" possa essere utilizzato con valori diversi da "code".

Per me "usare"

Reply:
Ok per me "usare"

Reply:
Ritengo che quanto riportato nella nota sia misleading. Il supporto ad applicazioni mobili credo sia out of scope. Qui consideriamo solo client confidential e di tipo web. Il supporto mobile nativo va indirizzato correttamente e non tramite una nota messa qui.